library/src/main/java/com/doctoror/rxcursorloader/RxCursorLoaderFlowableFactory.java

```diff
@@ -79,7 +77,8 @@ public void run() {
         @NonNull
         private final Scheduler mScheduler;
 
-        private Handler mHandler;
+        @NonNull
+        private final Handler mHandler = new Handler(Looper.getMainLooper());
```

Comment:
I'm not sure if I like this idea. We save one thread, but we still waste main thread for observer notification.
True that it's not a big problem since it's only used for handling a callback though.

Reply:
It saves N amount of threads + their startup time costs where N is the amount of flowables created.

Reply:
Handling the call back is essentially a co-op in this case